packages/react/react/src/hooks/useForceUpdate.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { useCallback, useRef } from 'react';
+import { useForceUpdate } from './useForceUpdate';
+
+function TestComponent() {
+  const forceUpdate = useForceUpdate();
+
+  const testRef = useRef<string>('토스');
+
+  const onClickTestRefUpdate = useCallback(() => {
+    testRef.current = 'toss';
+    forceUpdate();
+  }, [forceUpdate]);
+
+  return (
+    <div>
+      <button onClick={onClickTestRefUpdate}>{testRef.current}</button>
+    </div>
+  );
+}
+
+describe('forceUpdate를 호출하여 DOM 업데이트 확인', () => {
+  it('forceUpdate함수 실행 시 DOM이 업데이트 되었는가?', () => {
```

Comment:
```suggestion
describe('forceUpdate', () => {
  it('force-rerenders the component', () => {
```

We write test codes in English; what about rewriting them like this?

Reply:
GOOOOOD!!! Thanks : )